### DISCUSSION TITLE
Choose achievements I want to be shown on my profile

### DISCUSSION THREAD

**Discussion by Oleksii14** [community/community#56340] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey! Is there a way to choose achievements I want to be shown on my profile?

**Comment by radar-blip** [comment, marked as answer]
Yes. You can hide or show which ones will appear.
https://github.com/Oleksii14?tab=achievements

![hide](https://github.com/community/community/assets/39196767/fec2e898-296a-495b-9835-37ae061dcf3c)

**Comment by ElyasShamal** [comment]
Yes you can hide and unhide your Achievements
